src/storyshots.test.js

```diff
@@ -1,78 +0,0 @@
-import initStoryshots, {Stories2SnapsConverter} from '@storybook/addon-storyshots';
```

Comment:
If we remove these tests, how are we replacing them with the new packages?

Reply:
We are using test-runner now

Reply:
There should be some equivalent tests if we are removing this, right?